client/odie/query/index.ts

```diff
@@ -55,18 +55,24 @@ export const useOdieSendMessage = (): UseMutationResult<
 			return buildSendChatMessage( message, botNameSlug, chat.chat_id );
 		},
 		onSuccess: ( data ) => {
-			setChat( { ...chat, chat_id: parseInt( data.chat_id ) } );
+			setChat( { messages: chat.messages, chat_id: parseInt( data.chat_id ) } );
 			setOdieStorage( 'chat_id', data.chat_id );
 		},
 	} );
 };
 
 const buildGetChatMessage = (
 	botNameSlug: OdieAllowedBots,
-	chat_id: number | null | undefined
+	chat_id: number | null | undefined,
+	page: number,
+	perPage: number
 ): Promise< Chat > => {
-	const baseApiPath = `/help-center/odie/chat/${ botNameSlug }/${ chat_id }`;
-	const wpcomBaseApiPath = `/odie/chat/${ botNameSlug }/${ chat_id }`;
+	const urlQueryParams = new URLSearchParams( {
+		page_number: page.toString(),
+		items_per_page: perPage.toString(),
+	} );
+	const baseApiPath = `/help-center/odie/chat/${ botNameSlug }/${ chat_id }?${ urlQueryParams.toString() }`;
```

Comment:
Do we need this `/help-center/...` API path? I _think_ this had something to do with our previous attempt of putting the Odie client in Jetpack for atomic sites. If you look in [this file](https://github.com/Automattic/wp-calypso/blob/trunk/apps/editing-toolkit/editing-toolkit-plugin/help-center/class-wp-rest-help-center-odie.php), this route doesn't actually exist.